src/components/table/columns.ts

```diff
@@ -34,6 +38,27 @@ export class ColumnDefinitionFactory {
     }
 }
 
+export const formatTitle = (column: Column) => (): string | HTMLElement => {
+    const element = document.createElement('div');
+    element.innerText = column.title;
+
+    const icon = document.createElement('limel-icon');
+    icon.setAttribute('name', column.menuIcon);
+    icon.setAttribute('size', 'x-small');
+    icon.setAttribute('class', 'tabulator-header-menu-button');
+    icon.setAttribute('style', 'padding: 0 0 0 5px;');
+
+    element.appendChild(icon);
+    icon.addEventListener('click', headerClickListener);
+
+    return element;
+};
+
+function headerClickListener(event: MouseEvent | KeyboardEvent) {
+    event.stopPropagation();
```

Comment:
We will configure the event when we get this https://github.com/Lundalogik/crm-feature/issues/1748 done

Reply:
I don't think we need to wait for that PR in order to emit an event. They should be independent. A consumer _might_ use the popover to display something, but they might also do something completely different